### ISSUE TITLE
Elements alignment on 1/3 sized grid

### ISSUE THREAD

**Issue by cpBurn** [metafizzy/packery#447]
Hey guys!
I have a grid system that follows as such (big 66.666% and small 33.333% width) :
![grid](https://user-images.githubusercontent.com/4061253/27993204-651f6780-64a5-11e7-9912-77ef2e8c7d7c.png)

I group the the big tile with some small tiles around so it generates the way I want. The thing now, is that'd like to have the big one to be sometimes aligned to the right...and it becomes a bit of problem.

I tried to use `fit()` to push my tile to the right, but the small tiles are shifted down leaving a blank spaces

![grid2](https://user-images.githubusercontent.com/4061253/27993224-f56e9a72-64a5-11e7-8708-127fef9a415e.png)
 
I'd like to fill the blank space with a small tile...although I can programmatically do that, the rest of the layout remains a bit broken :/
Now, I don't reckon there's an easy way to achieve what I want, but maybe some of you might have some ideas?

**Comment by desandro** [comment]
I'm sorry to see you're having trouble with Packery. Could you provide a **reduced test case**? See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/packery/blob/master/.github/contributing.md#submitting-issues).

**Comment by cpBurn** [comment]
Right, here you https://codepen.io/anon/pen/dRayJg
It's basically the same as I have, except I have some filter before render the items in the order I put in the example :)

**Comment by desandro** [comment]
Thanks for that. I'm not sure this test case shows the bug that's occurring. I wasn't able to produce any gaps.

**Comment by cpBurn** [comment]
@desandro sorry for not being clear, the example was my regular grid. I just added a call to `fit` function to replicate the issue. As soon as you load you should see some gaps.

**Comment by desandro** [comment]
Ah, Thanks for that. I believe this may be a 1px rounding error issue. I'll have to take another look after #366 has been address.

**Comment by cpBurn** [comment]
@desandro cool thanks. Worth mentioning that if I change the window a bit (so it triggers resize), the grid renders correctly...maybe it's not recalculating the grid after `fit` call.

**Comment by cpBurn** [comment]
@desandro should expect a fix soon? Is there anything you reckon I could try to fix this issue?

**Comment by desandro** [comment]
Sorry, no fix soon. The 1px rounding issue is especially tricky. Packery works well by rounding pixel values, like for dragging. I'm afraid I won't be able to resolve this issue without breaking something else.

**Comment by belakm** [comment]
the only solution i found to combat this was to use 32 instead of 33 % and 65 instead of 66 %

**Comment by desandro** [comment]
Closing as a duplicate of #366. Subscribe to that issue for updates.
